app/views/base/layout.scala

```diff
@@ -330,7 +341,7 @@ object layout:
           dataBoardTheme   := pref.currentTheme.name,
           dataPieceSet     := pref.currentPieceSet.name,
           dataAnnounce     := lila.api.AnnounceStore.get.map(a => safeJsonValue(a.json)),
-          style            := zoomable.option(s"--zoom:$pageZoom")
+          style            := zoomable.option(s"---zoom:$pageZoom")
```

Comment:
see here that all of our existing css variables get an additional hyphen which forces them above the many colors in the browser dev tools property sort order.